src/snowflake/cli/app/telemetry.py

```diff
@@ -117,3 +120,15 @@ def log_command_usage():
 @ignore_exceptions()
 def flush_telemetry():
     _telemetry.flush()
+
+
+def check_feature_flags_usage():
+    from snowflake.cli.api.feature_flags import FeatureFlagMixin
+
+    flags = {}
+    for flag_cls in FeatureFlagMixin.__subclasses__():
+        # To skip test classes
+        if flag_cls.__name__.startswith("_"):
+            continue
+        flags.update({f.name: f.is_enabled() for f in flag_cls})
```

Comment:
Can't we print it similarly in `snow --info`? Now the reader has to know what is the default value of the feature flag to be sure how our CLI is configured.

Reply:
Very good point, the purpose is different but it's worth knowing if the value was set or not. Adjusted the code.

Reply:
Hmmm, the last changes in telemetry are good but my comment was more about info presented in `snow --info` that we can print there the same info as we send in telemetry and I think you didn't change anything there.

Reply:
Correct. The purpose is different. In telemetry we want to know what was really happening during execution time. In `--info` we want to se what users have in configuration. Telemetry approach scans all flags. Info approach works only on flags known to users. This is deliberate decision to avoid exposing all feature flags to users in case of using --info.